booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java

```diff
@@ -320,21 +322,25 @@ private static JsonNode getShipmentLocationTypeCode(JsonNode body, @NonNull Stri
       .orElse(null);
   }
 
-  private static final JsonContentCheck REQUESTED_CHANGES_PRESENCE = JsonAttribute.customValidator(
-    "Requested changes must be present for the selected Booking Status ",
-    body -> {
-      var bookingStatus = body.path("bookingStatus").asText("");
-      var issues = new LinkedHashSet<String>();
-      if (PENDING_CHANGES_STATES.contains(BookingState.fromString(bookingStatus))) {
-        var feedbacks = body.get("feedbacks");
-        if (feedbacks == null) {
-          issues.add("feedbacks is missing in allowed booking states %s".formatted(PENDING_CHANGES_STATES));
-        }
-      }
-      return issues;
-    }
-  );
-
+  static final JsonContentCheck FEEDBACK_PRESENCE =
```

Comment:
```suggestion
  static final JsonContentCheck FEEDBACKS_PRESENCE =
```
I'd use everywhere the same plural form, since that's the object name.